odbc/src/api/definitions.rs

```diff
@@ -110,3 +110,256 @@ pub enum UseBookmarks {
 
 #[derive(Clone, Copy, Debug)]
 pub enum AsyncStmtEvent {}
+
+#[repr(u16)]
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum InfoType {
```

Comment:
The `odbc_sys::InfoType` enum does not contain many of the variants we need. I created this enum with exactly the set of `InfoType`s we need to support for now.